consts.js

```diff
@@ -348,6 +348,9 @@ var thirteenStrings = [
     "paci", // lojban
     "ishumi nantathu", // isiZulu
     "lishumi elinesithathu" // isiXhosa
+    ,"muoi ba" // Vietnam
```

Comment:
Google says this translates to thirty three https://translate.google.com/?sl=vi&tl=en&text=muoi%20ba&op=translate

ಠ_ಠ

Reply:
google translate is not always correct, lol. BTW, you should add "mười ba" also

Reply:
Now it's translating to "father's day" lol. But when I do mười ba like you said it's 13. 👍